Gemfile.lock

```diff
@@ -0,0 +1,45 @@
+PATH
```

Comment:
Without a lockfile, there's no guarantee of which gem versions are installed, meaning people may see different failures.